edge-functions/image-response/next-env.d.ts

```diff
@@ -1,5 +1,4 @@
 /// <reference types="next" />
-/// <reference types="next/types/global" />
```

Comment:
Interesting - should this be removed?

Reply:
Yes, this file has been automatically updated by Next.js, so it is safe to delete that line. We removed the line from all recent example changes so it seems consistent.